### DISCUSSION TITLE
How to calculate morphometrics on masks not generated by AxonDeepSeg

### DISCUSSION THREAD

**Discussion by mathieuboudreau** [neuropoly/axondeepseg#443] [Help needed] [answered]
I'm analyzing external data, and some masks were provided for myelin and axons. I'd like to see how the morphometrics perform under these conditions (likely not going to do well), but I can't seem to figure out how. I assumed if I saved the files with the same convention (-myelin.png and -axon.png) and loaded them to ADS it would work, but it doesn't.

<img width="1128" alt="Capture d’écran 2021-01-26 à 09 40 35" src="https://user-images.githubusercontent.com/1421029/105849139-90090c80-5fba-11eb-90c0-2ae76d6448db.png">

@Stoyan-I-A any guidance?

**Comment by Stoyan-I-A** [comment, marked as answer]
@mathieuboudreau How did you load these images? It seems like you used FSLeyes' built-in image loader (judging by the data type of the files), which doesn't work with the plugin. 
To use these masks, you would have to create one image with both the axon and the myelin masks and use `Load existing mask` button.
You could also load them using the `Load PNG or TIF file` (the name should be changed because I believe it already supports more formats) and change the values of the mask using FSLeyes' python shell.
